tests/static_analyzer/test_lsp_client.py

```
@@ -0,0 +1,615 @@
+                    "message": "Unused import",
+                    "severity": 2,
+                    "code": "W0611",
+                }
+            ],
+        }
+
+        client._handle_notification("textDocument/publishDiagnostics", params)
+
+        diags = client.get_collected_diagnostics()
+        assert len(diags) == 1
+        file_key = list(diags.keys())[0]
```

Ruff (0.15.6)

[warning] Prefer `next(iter(diags.keys()))` over single element slice

Replace with `next(iter(diags.keys()))`

(RUF015)